engine/src/main/java/com/odysseusinc/arachne/executionengine/api/v1/AnalysisController.java

```diff
@@ -145,7 +149,7 @@ public ResponseEntity<MultiValueMap<String, Object>> analyzeSync(
 
         Long id = analysisRequest.getId();
         log.info("Started processing request for synchronous analysis ID = {}", id);
-        final File analysisDir = AnalisysUtils.extractFiles(files, false);
+        final File analysisDir = AnalisysUtils.extractFiles(files, analysisParentDir, false);
```

Comment:
Without having a relevant client that uses it, I'd prefer to keep changes to a minimum.
However I'm not sure about deprecating it. I think it still could be useful for datasource check which we need to fix on the datanode side. Since database network host visibility from execution container will be quite a common configuration issue, there needs to be a convenient way to diagnose it.